### PR TITLE
adding npm task for runing tests in with debugging enabled

### PR DESCRIPTION
**Pull request by MiguelCastillo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "description": "Tell Slack you're working from home today 🏡",
   "repository": {
-	  "type": "git",
-	  "url": "https://github.com/Symphono/wfh-slack-command"
+    "type": "git",
+    "url": "https://github.com/Symphono/wfh-slack-command"
   },
   "engine": {
     "node": "7.6.0"
```
Comment (MiguelCastillo) [inline package.json:10]: right here we define node 7 :)

File: package.json [hunk 2]
```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "description": "Tell Slack you're working from home today 🏡",
   "repository": {
-	  "type": "git",
```
Comment (MarkLeMerise) [inline package.json:6]: What's going on with this whitespace?

File: package.json [hunk 3]
```diff
@@ -3,16 +3,17 @@
...
+    "url": "https://github.com/Symphono/wfh-slack-command"
   },
   "engine": {
     "node": "7.6.0"
   },
   "scripts": {
     "start": "node src/index.js",
     "test-watch": "mocha --debug-brk --inspect --watch",
-	"test": "mocha"
+    "test-debug": "mocha --inspect-brk",
```
Comment (MarkLeMerise) [inline package.json:15]: Do you think we should just get rid of the `test-watch` command, then? Or maybe add the `--watch` flag here?

File: package.json (hunk 1 above)
Comment (MarkLeMerise) [reply package.json:10]: Sorry, I don't know why I thought it was Node 4.

File: package.json (hunk 2 above)
Comment (MiguelCastillo) [reply package.json:6]: funny you ask... the file i cloned had tabs and spaces mixed in... i changed the tabs that were out of place to regular spaces.  let me double check to make sure i didnt mess it up.

File: package.json (hunk 3 above)
Comment (MiguelCastillo) [reply package.json:15]: well - it depends.  i think the ability for watch is workflow dependent.  when you do your debugging, do you keep the process running with watch?  I guess we can just get rid of `test-debug`, which is a one time debugging session.

File: package.json (hunk 2 above)
Comment (MarkLeMerise) [reply package.json:6]: Oh, you know what, I think I forgot add a rule to the `.editorconfig` for `package.json` as a 2-space file. Would you mind doing that?

Comment (MiguelCastillo) [reply package.json:6]: sure - separate PR.  @TNagaraja can probably pick that up as a task?

Comment (TNagaraja) [reply package.json:6]: Already in pending PR to master

**Pull request merged**